src/openfl/display/DisplayObject.hx

```diff
@@ -1325,26 +1320,92 @@ class DisplayObject extends EventDispatcher implements IBitmapDrawable #if openf
 	}
 	
 	
-	public function __updateChildren (transformOnly:Bool):Void {
+	private function __updateRenderData ():Void {
 		
 		var renderParent = __renderParent != null ? __renderParent : parent;
+		if (__isMask && renderParent == null) renderParent = __maskTarget;
```

Comment:
I know this line was copied, but I was actually thinking why the render parent is the mask target if there is no other render parent. This is something we don't do in other places (there we simply have the single line above). Do you know that?